src/panels/lovelace/editor/view-editor/hui-view-visibility-editor.ts

```diff
@@ -0,0 +1,121 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  property,
+  CSSResult,
+  css,
+} from "lit-element";
+import "@polymer/paper-input/paper-input";
+
+import { fireEvent } from "../../../../common/dom/fire_event";
+import { configElementStyle } from "../config-elements/config-elements-style";
+import { ShowViewConfig } from "../../../../data/lovelace";
+
+import "../../components/hui-theme-select-editor";
+
+import { User } from "../../../../data/user";
+import memoizeOne from "memoize-one";
+import { compare } from "../../../../common/string/compare";
+import { HaSwitch } from "../../../../components/ha-switch";
+
+declare global {
+  interface HASSDomEvents {
+    "view-visibility-changed": {
+      visible: ShowViewConfig[];
+    };
+  }
+}
+
+@customElement("hui-view-visibility-editor")
+export class HuiViewVisibilityEditor extends LitElement {
+  static get styles(): CSSResult {
+    return css`
+      .panel {
+        color: var(--secondary-text-color);
+      }
+      .flex {
+        flex: 1;
+        margin: 0 16px;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        min-width: 0;
+      }
+    `;
+  }
+  @property() public users!: User[];
+  @property() public visible!: ShowViewConfig[];
+
+  private _sortedUsers = memoizeOne((users?: User[]) => {
+    if (!users) {
+      return [];
+    }
```

Comment:
Instead of handling this here, check if `users` is defined in `render`

Reply:
done